packages/internals/src/sendPanic.ts

```diff
@@ -20,7 +19,20 @@ const debug = Debug('prisma:sendPanic')
 // cleanup the temporary files even when an uncaught exception occurs
 tmp.setGracefulCleanup()
 
-export async function sendPanic(error: RustPanic, cliVersion: string, engineVersion: string): Promise<number> {
+type SendPanic = {
+  error: RustPanic
+  cliVersion: string
+  enginesVersion: string
+
+  // retrieve the database version for the given schema or url, without throwing any error
+  getDatabaseVersionSafe: (schemaOrUrl: string) => Promise<string | undefined>
+}
+export async function sendPanic({
+  error,
+  cliVersion,
+  enginesVersion,
+  getDatabaseVersionSafe,
+}: SendPanic): Promise<number> {
```

Comment:
Turning a list of arguments to an object with named properties for ease of use. It'd be easy passing the arguments in the wrong sequence instead.